2022-08-08 11:08:16,062 INFO    Thread-33 :1806750 [wandb_setup.py:_flush():76] Configure stats pid to 1806750
2022-08-08 11:08:16,062 INFO    Thread-33 :1806750 [wandb_setup.py:_flush():76] Loading settings from /home/kana/.config/wandb/settings
2022-08-08 11:08:16,062 INFO    Thread-33 :1806750 [wandb_setup.py:_flush():76] Loading settings from wandb/settings
2022-08-08 11:08:16,063 INFO    Thread-33 :1806750 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'noisy_label_cleaning_eval', 'entity': 'kka-na', 'root_dir': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script', 'run_id': 'mr1ndgk0', 'sweep_param_path': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/sweep-zppdxpuh/config-mr1ndgk0.yaml', 'sweep_id': 'zppdxpuh'}
2022-08-08 11:08:16,063 WARNING Thread-33 :1806750 [wandb_setup.py:_flush():76] Could not save program above cwd: ../src/hyp_tuning.py
2022-08-08 11:08:16,063 INFO    Thread-33 :1806750 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program': '../src/hyp_tuning.py'}
2022-08-08 11:08:16,063 INFO    Thread-33 :1806750 [wandb_init.py:_log_setup():461] Logging user logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220808_110816-mr1ndgk0/logs/debug.log
2022-08-08 11:08:16,063 INFO    Thread-33 :1806750 [wandb_init.py:_log_setup():462] Logging internal logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220808_110816-mr1ndgk0/logs/debug-internal.log
2022-08-08 11:08:16,067 INFO    Thread-33 :1806750 [wandb_init.py:init():495] calling init triggers
2022-08-08 11:08:16,067 INFO    Thread-33 :1806750 [wandb_init.py:init():498] wandb.init called with sweep_config: {'BATCH_SIZE_PER_IMAGE': 32, 'IMS_PER_BATCH': 2, 'iteration': 1500, 'learning_rate': 0.0009497608026941012, 'model': 'COCO-Detection/rpn_R_50_FPN_1x.yaml'}
config: {}
2022-08-08 11:08:16,070 INFO    Thread-33 :1806750 [wandb_init.py:init():548] starting backend
2022-08-08 11:08:16,070 INFO    Thread-33 :1806750 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-08 11:08:16,080 INFO    Thread-33 :1806750 [backend.py:ensure_launched():217] starting backend process...
2022-08-08 11:08:16,117 INFO    Thread-33 :1806750 [backend.py:ensure_launched():222] started backend process with pid: 1819405
2022-08-08 11:08:16,118 INFO    Thread-33 :1806750 [wandb_init.py:init():558] backend started and connected
2022-08-08 11:08:16,132 INFO    Thread-33 :1806750 [wandb_run.py:_config_callback():1131] config_cb None None {'BATCH_SIZE_PER_IMAGE': 32, 'IMS_PER_BATCH': 2, 'iteration': 1500, 'learning_rate': 0.0009497608026941012, 'model': 'COCO-Detection/rpn_R_50_FPN_1x.yaml'}
2022-08-08 11:08:16,137 INFO    Thread-33 :1806750 [wandb_init.py:init():634] updated telemetry
2022-08-08 11:08:16,138 INFO    Thread-33 :1806750 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2022-08-08 11:08:16,990 INFO    Thread-33 :1806750 [wandb_run.py:_on_init():1919] communicating current version
2022-08-08 11:08:17,031 INFO    Thread-33 :1806750 [wandb_run.py:_on_init():1923] got version response upgrade_message: "wandb version 0.13.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-08-08 11:08:17,032 INFO    Thread-33 :1806750 [wandb_init.py:init():700] starting run threads in backend
2022-08-08 11:08:18,134 INFO    Thread-33 :1806750 [wandb_run.py:_console_start():1893] atexit reg
2022-08-08 11:08:18,138 INFO    Thread-33 :1806750 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2022-08-08 11:08:18,138 INFO    Thread-33 :1806750 [wandb_run.py:_redirect():1771] Redirecting console.
2022-08-08 11:08:18,139 ERROR   Thread-33 :1806750 [wandb_run.py:_redirect():1830] Failed to redirect.
Traceback (most recent call last):
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1823, in _redirect
    out_redir.install()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 702, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 652, in add_fd
    self._register()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 637, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-08-08 11:08:18,140 INFO    Thread-33 :1806750 [wandb_init.py:init():732] run started, returning control to user process
2022-08-08 11:15:08,992 INFO    Thread-33 :1806750 [wandb_run.py:_finish():1680] finishing run kka-na/noisy_label_cleaning_eval/mr1ndgk0
2022-08-08 11:15:08,993 INFO    Thread-33 :1806750 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 0
2022-08-08 11:15:08,997 INFO    Thread-33 :1806750 [wandb_run.py:_restore():1834] restore
2022-08-08 11:15:09,003 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 530
  total_bytes: 530
}

2022-08-08 11:15:09,157 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 530
  total_bytes: 530
}

2022-08-08 11:15:09,420 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 530
  total_bytes: 530
}

2022-08-08 11:15:10,211 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 530
  total_bytes: 6039
}

2022-08-08 11:15:10,313 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 530
  total_bytes: 6039
}

2022-08-08 11:15:10,416 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 530
  total_bytes: 6039
}

2022-08-08 11:15:10,519 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 530
  total_bytes: 6039
}

2022-08-08 11:15:10,621 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 530
  total_bytes: 6039
}

2022-08-08 11:15:10,724 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3772
  total_bytes: 6039
}

2022-08-08 11:15:10,826 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3772
  total_bytes: 6039
}

2022-08-08 11:15:10,929 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3772
  total_bytes: 6039
}

2022-08-08 11:15:11,031 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3772
  total_bytes: 6039
}

2022-08-08 11:15:11,134 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3772
  total_bytes: 6039
}

2022-08-08 11:15:11,237 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-08 11:15:11,339 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-08 11:15:11,442 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-08 11:15:11,544 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-08 11:15:11,647 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-08 11:15:11,749 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-08 11:15:12,998 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-08 11:15:13,980 INFO    Thread-33 :1806750 [wandb_run.py:_on_finish():1991] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}
local_info {
}

2022-08-08 11:15:15,119 INFO    Thread-33 :1806750 [wandb_run.py:_footer_history_summary_info():3099] rendering history
2022-08-08 11:15:15,120 INFO    Thread-33 :1806750 [wandb_run.py:_footer_history_summary_info():3131] rendering summary
2022-08-08 11:15:15,125 INFO    Thread-33 :1806750 [wandb_run.py:_footer_sync_info():3055] logging synced files
